BUILDING_ON_WINDOWS.md

```diff
@@ -101,14 +101,14 @@ Build results will be placed in a folder at the same level as the "chatterino2"
 
 If you build chatterino, the result directories will contain a `chatterino.exe` file in the `$OUTPUTDIR\release\` directory. This `.exe` file will not directly run on any given target system, because it will be lacking various Qt runtimes.
 
-To produce a standalone package, you need to generate all required files using the tool `windeployqt`. This tool can be found in the `bin` directory of your Qt installation, e.g. at `C:\Qt\5.15.2\msvc2019_64\bin\windeployqt.exe`.
+To produce a standalone package, you will need to generate all required files by using the tool `windeployqt`. This tool can be found in the `bin` directory of your Qt installation, e.g. at `C:\Qt\5.15.2\msvc2019_64\bin\windeployqt.exe`.
```

Comment:
I don't see the upside of this rephrasing

CC @Felanbird

Reply:
> I don't see the upside of this rephrasing
> 
> CC @Felanbird

The change is just present tense vs future tense. 

If I was explaining to somebody what steps they `will` need to take in the future for building. The old variation being present tense, for reading what you steps you should take right now. 

I'm not sure what variation would be better personally.

Reply:
```suggestion
To produce a standalone package, you need to generate all required files using the tool `windeployqt`. This tool can be found in the `bin` directory of your Qt installation, e.g. at `C:\Qt\5.15.2\msvc2019_64\bin\windeployqt.exe`.
```

Let's go back to present tense then